apps/core/src/utils/stake/createValidationSchema.ts

```diff
@@ -38,6 +41,13 @@ export function createValidationSchema(
                         message: 'Insufficient funds',
                     });
                 }
+
+                const canStake = availableBalance >= minimumStake;
+                if (!canStake)
+                    return ctx.createError({
+                        message: `Insufficient funds to stake a minimum of ${MIN_NUMBER_IOTA_TO_STAKE} ${coinSymbol}`,
+                    });
```

Comment:
looks like this can be removed because this case is covered in the 'min' test on line 34 > amount ? `amount.shiftedBy(decimals).gte(minimumStake.toString()) : false,`

Reply:
`amount` in line 34 is the user input, not the availableBalance of the wallet, that's why it's different